pkg/gameserverallocations/allocation_cache.go

```diff
@@ -237,14 +237,14 @@ func (c *AllocationCache) ListSortedGameServers(gsa *allocationv1.GameServerAllo
 			for _, priority := range gsa.Spec.Priorities {
 				res := compareGameServers(&priority, gs1, gs2)
 				switch priority.Order {
-				case allocationv1.GameServerAllocationAscending:
+				case agonesv1.GameServerPriorityAscending:
```

Comment:
Suggestion: Pull the renaming out as a separate PR, just to remove some changes from this one.

Reply:
I'm going to leave this in since I changes the location of the code to agonesv1 this variable would have changed regardless.